portfolio/src/main/webapp/index.html

```diff
@@ -7,9 +7,20 @@
     <script src="script.js"></script>
   </head>
   <body>
+
+    <form action="/form-handler" method="POST">
+        <p>Enter some text here:</p>
+        <textarea name="text-input">hello world</textarea>
+        <br/>
+        <input type="submit" />
+      </form>
+      
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Erica Chen</h1>
       <p>This is my portfolio.</p>
+      <img src="/images/pfp.PNG" 
+      width="200" 
+      height="400"/>
```

Comment:
Stretch goal: Use your CSS file to create a class (with this style) that you can apply to _all_ images.